Relax tight dependencies that upstream uses to keep MSRV low

Upstream is concerned with keeping the MSRV low, but we don't need to
commit to that same approach to resolving the concern (debian archive
management should handle it for us).

Furthermore, sequoia-openpgp should also work with base64 version
0.12.*, not just 0.11.*.

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

--- a/Cargo.toml	2020-04-24 15:13:48.609357466 -0400
+++ b/Cargo.toml	2020-04-24 15:13:59.710831392 -0400
@@ -32,7 +32,7 @@
 version = "1"
 
 [dependencies.base64]
-version = "0.11"
+version = ">= 0.11, < 0.13"
 
 [dependencies.buffered-reader]
 version = "0.16"
@@ -79,9 +79,9 @@
 version = "1"
 
 [dependencies.unicode-normalization]
-version = "= 0.1.9"
+version = "0.1.9"
 [dev-dependencies.rpassword]
-version = "=4.0.3"
+version = "4.0.3"
 [build-dependencies.lalrpop]
 version = "0.17"
 
